Sources/Testing/Running/Runner.RuntimeState.swift

```diff
@@ -47,9 +47,19 @@ extension Runner {
       return
     }
 
-    configuration.eventHandler = { [eventHandler = configuration.eventHandler] event, context in
+    configuration.eventHandler = { [oldEventHandler = configuration.eventHandler] event, context in
```

Comment:
This code isn't new, but needed to move from `handleEvent` as it imposed the wrong layering (saving the attachment before any event handlers actually ran.)